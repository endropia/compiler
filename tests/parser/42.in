var b: string;

begin

b := 'hello';
 writeln(b);
end.
