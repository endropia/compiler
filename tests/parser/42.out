program : Unnamed program
   var: 
      b
      type: string
   stmts:
      :=
         b
         hello
      call
         writeln
            b
